### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by Amephest**
Organized ## Table of Contents
Added explanations to ## Official Meetings
Categorized ## Use Your Dai, expanded on platform details.

### CODE REVIEW

**Comment by Davidutro** [comment]
FIX YO SHIT

*Headers

**Comment by Davidutro** [comment]
fixed conflict (added CDP.FYI)

File: README.md [hunk 1]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+- [**Celcius**](https://celsius.network/): Centralized Mobile app for Crypto and DAI lending. 
+
+# MakerDAO Metrics and Statistics
+- [**Mkr.tools**](https://mkr.tools): MakerDAO System Explorer and Informational Statistics
+- [**Makerscan**](https://makerscan.io/): MakerDAO System Explorer
+- [**Vishesh's Graphs**](http://makerdao.descipher.io/): Data representations hosted by Vishesh Choudhry
+- [**daiembassy**](https://www.daiembassy.online/): MakerDAO System Explorer
+- [**State of the Dapps: MakerDAO**](https://www.stateofthedapps.com/dapps/makerdao): MakerDAO Stats
+- [**Supermax**](https://playground.supermax.cool/makerdao): MakerDAO Stats
+- [**Vishesh's Graphs**](http://makerdao.descipher.io/): Data representations hosted by Vishesh Choudhry
```
Comment (ghettodev) [inline README.md]: this is in the list above

File: README.md [hunk 2]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+- [**MakerDAO Stability Fee**](https://makerstabilityfee.com/): Dashboard to view current stability fee and a sign up to be notified when changed
+
+## DAI Stats
+- [**DaiStats**](https://daistats.com/): Simple Dai Statistics by Mariano Conti
+- [**DaiStats**](https://dai-in-defi.com/): Supply and Demand Dashboard for DAI
```
Comment (ghettodev) [inline README.md]: this should be dai-in-defi

File: README.md [hunk 3]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+- [**Nomics: Dai**](https://nomics.com/assets/dai-dai#statistics): Dai Statistics through an API aggregator
+
+## DAI Price
+- [**Daiprice**](https://www.daiprice.info/): Dashboard to view Dai USD price and other metrics 
+- [**DaiPeg**](https://daipeg.com/): Interactive Dashboard to view Dai USD price, calculated accross several exchanges
+- [**dai.stablecoin.science**](https://dai.stablecoin.science/): Dashboard to view Dai USD price
+- [**Dai for sale on Coinbase below $1**](https://dai.defi.ai/): Visual & Numerical representation of the orderbook below $1
```
Comment (ghettodev) [inline README.md]: this is for above and below order book. it changes

File: README.md [hunk 4]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+- [**Tracker of 24hr VWAP of Dai**](http://dai.descipher.io/): Dai Price Tracker by Vishesh Choudhry
+
+## Ecosystem Statistics and Metrics
+- [**DefiPulse**](https://defipulse.com/): Get a sense for the Defi space in general 
+- [**DAppTotal**](https://dapptotal.com/defi): Defi Rankings
+- [**Defitracker**](https://defitracker.io/): Dashboard to track view Dai across DeFi protocols, including Compound, Dydx, Uniswap Pools and more.
```
Comment (ghettodev) [inline README.md]: this is a personal defi tracker, not ecosystem

File: README.md [hunk 5]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+
+## Stablecoin Comparisons
+- [**TokenAnalyst: Stablecoins**](https://www.tokenanalyst.io/stablecoins): Stablecoin Comparison
+- [**TRMLabs Insights**](https://insights.trmlabs.com/?tokens=DAI,USDT,TUSD,GUSD): Stablecoin Comparison
+- [**Loanscan**](https://loanscan.io/): Financial information and analytics for debt issued on the Ethereum blockchain
```
Comment (ghettodev) [inline README.md]: this isn't really a stablecoin tool, it's decentralized lending

File: README.md [hunk 6]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+
+## Noncustodial Exchanges
+- [**Eth2Dai**](https://eth2dai.com/exchange/WETH/DAI)
+- [**Radar Relay**](https://app.radarrelay.com/WETH/DAI)
+- [**Totle**](https://app.totle.com)
+- [**DDex**](https://ddex.io/trade/WETH-DAI)
+- [**CoinChangeX**](https://www.coinchangex.com/#!/trade/DAI-ETH)
+- [**ForkDelta**](https://forkdelta.app/#!/trade/DAI-ETH)
+- [**Paradex**](https://paradex.io/market/weth-dai)
+- [**coinected**](https://coinected.io)
```
Comment (ghettodev) [inline README.md]: capitalize

File: README.md [hunk 7]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+- [**Exmo**](https://exmo.com/en/trade#?pair=DAI_ETH): Based in England
+- [**Panda**](https://panda.exchange/en-IC/): Based in Latin America
+- [**Probit**](https://www.probit.com/en-us): Based in Republic of Seychelles
+- [**COSS.IO**](https://coss.io/): Based in Singapore
+
+# Spend Your Dai
+- [**NeedsList**](https://needslist.co/donate): Purchase items that are critically needed & help with disaster preparedness
+- [**SendCrypto**](https://sendcryptobot.io/): Tip with Dai on Twitter, Slack, Reddit, and Telegram
+- [**Ethichub**](https://ethichub.com/): Provide p2p Loans to Communities Around the Globe
+- [**Dharma**](https://dharma.io/): Loan out your Crypto, including Dai
```
Comment (ghettodev) [inline README.md]: lending

File: README.md [hunk 8]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+- [**TheBigCoin**](https://www.thebigcoin.io/pay-with-dai): Spend Dai Anywhere
+- [**Bidali**](https://giftcards.bidali.com/buy-giftcards-with-dai/): Buy gift cards with Dai
+- [**Wirex**](https://wirexapp.com/card/): Visa debit card that can be top-up with DAI 
+
+## Merchant Solutions
+- [**8x Protocol**](https://8xprotocol.com/)
+- [**Groundhog**](https://groundhog.network/)
```
Comment (ghettodev) [inline README.md]: this isn't launched. I'd like to avoid anything that isn't already live

File: README.md [hunk 9]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
...
+- [**Ink Protocol**](https://paywithink.com/)
+- [**Savvy/PayBear**](https://www.savvy.io/)
+
+## Open-Source payment processors 
+- [**Daipay**](https://github.com/codevet/daipay/)
```
Comment (ghettodev) [inline README.md]: this looks like a github PoC

File: README.md [hunk 10]
```diff
@@ -46,15 +46,13 @@ Below is a collection of resources for anyone interested in exploring MakerDao.
 ### DAI Use Cases
 - [Use Your Dai](#use-your-dai)
 - [Lend Your Dai](#lend-your-dai)
-- [Watch your Dai](#watch-your-dai)
```
Comment (ghettodev) [inline README.md]: why is this coming out?

File: README.md [hunk 11]
```diff
@@ -143,7 +141,168 @@ The scientific Governance and Risk Meetings is where the MakerDAO community disc
 - [FAQs: French](https://github.com/makerdao/community/tree/master/faqs/fr)
 - [FAQs: Korean](https://github.com/makerdao/community/tree/master/faqs/ko)
 
-## In Depth Analysis
+# Use Your Dai
+
+## All-In-One's
```
Comment (ghettodev) [inline README.md]: no convinced people will know what an all-in-one is

**Comment by ghettodev** [comment]
Assuming this is abandoned.

**Pull request closed without merging**